lib/services/consumed/datatip-provider.js

```diff
@@ -0,0 +1,85 @@
+/* @flow */
+
+import { Range, Point } from "atom";
+import React from "react";
+import { RichMedia, Media } from "@nteract/outputs";
+
+import { Markdown } from "./../../components/result-view/markdown";
+import { getRow } from "./../../code-manager";
+import store from "./../../store";
+import { js_idx_to_char_idx } from "./../../utils";
+
+class DatatipProvider {
+  providerName = "Hydrogen";
+  priority = 1;
```

Comment:
There should probably be a config to set the priority like autocomplete has. to do it dynamically you should be able to just use atom.config, but if not, I would make this into a getter function which returns the config

Reply:
Just saw your config goals, so I know this is on your todo. I am leaving it open so you know how to do it dynamically. If you need examples look at autocomplete